examples/iiot/example_client_iiot_plot_timing.py

```diff
@@ -0,0 +1,221 @@
+
+import itertools
+import os
+import pickle
+import numpy as np
+import matplotlib.pyplot as plt
+
+# do scatter plot of delays over rounds on exit
+# rounds = [*range(len(rtt_array))]
+# plt.scatter(rounds, rtt_array)
+# plt.ylabel("Round Trip Time [s]")
+# plt.xlabel("Round Number")
+# plt.show()
+
+# check for timing data written by actuator/client and display it
+print("checking for external statistic files...")
+statistics = {}
+for file in os.listdir("."):
+    if file.endswith(".pckl"):
+        print(f"loading {file}")
+        with open(file, 'rb') as handle:
+            dict = pickle.load(handle)
+        statistics[file]=dict
```

Comment:
```suggestion
            statistics[file] = pickle.load(handle)
```

I would not use 'dict' as variable name, as this is already a python type

Reply:
done